metaflow/decorators.py

```diff
@@ -217,7 +248,13 @@ def add_decorator_options(cmd):
                     % (deco.name, option, seen[option])
                 )
                 raise MetaflowInternalError(msg)
+            elif deco.name.lower() in existing_params:
+                raise MetaflowInternalError(
+                    "Flow decorator '%s' uses an option '%s' which is a reserved "
+                    "keyword. Please use a different option name." % (deco.name, option)
+                )
             else:
+                kwargs["envvar"] = "METAFLOW_FLOW_%s" % option.upper()
```

Comment:
there can only be one `envvar` in kwargs?

Reply:
This is click's way of indicating which env-var maps to that option. So yes, there is only one env-var that maps to the option.